sphinx/source/docs/user_guide/geo.rst

```diff
@@ -33,7 +33,8 @@ Google Maps
 Bokeh can also plot glyphs over a Google Map using the :func:`~bokeh.plotting.gmap.gmap`
 function. You must pass this function `Google API Key`_ in order for it to work, as
 well as any :class:`~bokeh.models.map_plots.GMapOptions` to configure the Google Map
-underlay.
+underlay. Though not revealed, the Google API Key passed above is stored as a JSON
+object for further functioning.
```

Comment:
```suggestion
underlay. The Google API Key will be stored in the Bokeh Document JSON. 
```

In a static, standalone document the JSON is plainly embedded, so we should not claim it is not revealed.